cmd/geth/main.go

```diff
@@ -81,6 +81,7 @@ var (
 		utils.EthashDatasetsInMemoryFlag,
 		utils.EthashDatasetsOnDiskFlag,
 		utils.EthashDatasetsLockMmapFlag,
+		utils.EthashEpochLengthFlag,
```

Comment:
Does this flag really want to be included here? If the flag is only for the `misc` commands?